.github/ISSUE_TEMPLATE/feature_template.md

```diff
@@ -18,10 +18,22 @@ If so, please describe the problem
 
 Include any specific requirements you need
 
+**Notify relevant community channels**
+
+Consider raising this topic on [Slack] in the #development channel or during a
+[community meeting] for awareness. List any relevant SIGs below from the
+[teams] list in the following form so that members of those teams are notified:
+
+- @cilium/example
```

Comment:
```suggestion
List any relevant SIGs below from the [teams] list in the following form so that members of those teams are notified:

- @cilium/example

If the SIG is not clear or you haven't gotten feedback, consider raising this topic on [Slack] in the #development channel or during a [community meeting] for awareness. 
```